### ISSUE TITLE
Questions about the training dataset

### ISSUE THREAD

**Issue by ranpin** [UX-Decoder/Segment-Everything-Everywhere-All-At-Once#63]
For panorama segmentation, I read in the paper that the dataset from coco2017 was used for training and the annotation for panorama segmentation was used and images from refcocog were excluded,the final total was 107k images. If I I understand correctly, I have the following two questions:

![image](https://github.com/UX-Decoder/Segment-Everything-Everywhere-All-At-Once/assets/51260021/86adf07d-e53a-479b-bbc2-ec8d5c28da1b)

The first question is: in the annotation of coco2017's dataset with panoramic segmentation, I don't find mask annotation, it seems to have only bbox information. Is this supposed to be the case, or am I looking for the wrong file? If so, how was the panorama segmentation done?

![image](https://github.com/UX-Decoder/Segment-Everything-Everywhere-All-At-Once/assets/51260021/8deaa5b5-6903-47a5-930c-d62f52195d8a)
![image](https://github.com/UX-Decoder/Segment-Everything-Everywhere-All-At-Once/assets/51260021/68bbf946-5e72-45d7-94d5-03edd4a11c0c)

Second question: according to the table below, the training uses roughly 0.2M of data, which doesn't seem to be consistent with the 107K mentioned earlier, which I don't quite understand. If the images in refcocog are not excluded,the training set of coco2017 and the training set of LVIS do add up to about 0.2M.

![image](https://github.com/UX-Decoder/Segment-Everything-Everywhere-All-At-Once/assets/51260021/eeba5147-a29a-49fc-ae04-dd32c79a63b7)

My main question in the end is for panorama segmentation what is your dataset, I don't understand it very well by looking at the paper and would appreciate an explanation, thank you.

**Comment by MaureenZOU** [comment]
Thanks so much for your interests in the work : ) 
1. For preparing the panoptic segmentation data, please refer to: https://github.com/facebookresearch/Mask2Former/blob/main/datasets/README.md
2. For LVIS and COCO, what I am doing is just replace the COCO mask to LVIS if their bbox overlap is > 0.7 (may not be the exact number). And the number 0.2M on Table1 is an honest mistake, we have already update the arXiv version.

![Screenshot 2023-08-04 at 11 05 15 AM](https://github.com/UX-Decoder/Segment-Everything-Everywhere-All-At-Once/assets/11957155/7539bdfc-9cdd-48c6-917b-ecbea93fe577)
